packages/tgpu-cli/tgpu-cli.mjs

```diff
@@ -23,17 +24,54 @@ const COMMANDS = {
       const input = args['--input'];
       const output = args['--output'];
 
-      if (!input || !output) {
+      if (!input) {
         console.error(
-          `${color.Red}Error: Missing required arguments: ${color.Yellow}--input${color.Reset} and ${color.Yellow}--output${color.Reset}`,
+          `${color.Red}Error: Missing required argument: ${color.Yellow}--input${color.Reset}`,
         );
         exit(1);
       }
 
-      try {
-        await generate(input, output);
-      } catch (error) {
-        console.error(`${color.Red}Error: ${error.message}${color.Reset}`);
+      const files = await glob(input);
+
+      if (files.length === 0) {
+        console.warn(
+          `${color.Yellow}Warning: No files found for pattern: "${input}"${color.Reset}`,
+        );
+        exit(0);
+      }
+
+      if (!output && !input.endsWith('.wgsl')) {
+        console.error(
+          `${color.Red}Error: No output name provided and provided input doesn't end with .wgsl ${color.Reset}`,
+        );
+        exit(1);
+      }
+
+      if (output && files.length > 1) {
+        if (!input.endsWith('.wgsl')) {
```

Comment:
What is this check for?

Reply:
oh, must be a copy-paste error. i'll remove it here